runtime/commonMain/src/kotlinx/serialization/json/JsonExceptions.kt

```diff
@@ -7,21 +7,57 @@ package kotlinx.serialization.json
 import kotlinx.serialization.*
 
 /**
- * Generic exception indicating a problem with JSON operations.
+ * Generic exception indicating a problem with JSON serialization.
```

Comment:
```suggestion
 * Generic exception indicating a problem with JSON serialization and deserialization.
```

Reply:
Btw, this is not _100%_ true since this exception is also thrown when `JsonElement` mismatches expected type (e.g. `element.jsonObject` call when `element is JsonArray`)

Reply:
well, it can be considered as the problem with deserialization :)

Reply:
It's more of a transformation...